README.md

```diff
@@ -29,89 +17,107 @@ Thank you to [Radek Tomšej](https://github.com/tomsej) for initial development
  </a>
 </p>
 
-List of [pre-commit](https://pre-commit.com) hooks to ensure the quality of your [dbt](https://www.getdbt.com) projects.
+## Sponsors
+
+<img src=".github/datacoves-logo.png" alt="Datacoves" width="150"/><br>
+The turnkey analytics stack, find out more at [datacoves.com](https://datacoves.com/).
+
+<br>
+
+<img style="margin-top: 20px;" src=".github/montreal-analytics-logo.png" alt="Montreal Analytics" width="150"/><br>
+[Montreal Analytics](https://www.montrealanalytics.com/?utm_source=blog&utm_medium=medium&utm_campaign=dbt-checkpoint) is a full-stack data consultancy servicing North America, and are both a dbt Preferred Consulting Partner and Platinum dbt Certification Award winner.
 
 ## Goal
 
-*Quick ensure the quality of your `dbt` projects*.
+[pre-commit](https://pre-commit.com) hooks to ensure the quality of your [dbt](https://www.getdbt.com) projects.
```

Comment:
dbt-checkpoint provides [pre-commit](https://pre-commit.com/) hooks to help you better govern yourr [dbt](https://www.getdbt.com/) projects.